python/src/aiconfig/scripts/rage/rage.py

```diff
@@ -0,0 +1,226 @@
+    """
+
+    # Split the command into shell-like syntax
+    cmd_parts = shlex.split(command)
+
+    # Try to execute the command and capture output
+    try:
+        process = subprocess.Popen(cmd_parts, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        stdout, stderr = process.communicate()
+
+        # Decode bytes to string and return formatted message
+        stdout_str = stdout.decode().strip()
+        stderr_str = stderr.decode().strip()
+
+        return Ok(f"\nSTDOUT:\n'{stdout_str}'\nSTDERR:\n'{stderr_str}'")
+
+    except Exception as e:
+        return core_utils.ErrWithTraceback(e)
+
+
+def _fmt_command_input_output(command: str, command_output: Result[str, str]) -> str:
+    command_output_str = command_output.unwrap_or("Couldn't run command :(. Please run manually.")
+    return dedent(
+        f"""
+            Command: {command}
+            Output: {command_output_str}
+            """
+    )
+
+
+def _create_issue_draft() -> Result[None, str]:
+    title = "Bug Report [TODO: brief description]"
+    server_log_path = _look_for_log("editor_flask_server.log")
+    aiconfig_log_path = _look_for_log("aiconfig.log")
+    server_log_str = f"see {server_log_path}" if server_log_path else "Couldn't find the log file :( Check your terminal output"
+    aiconfig_log_str = f"see {aiconfig_log_path}" if aiconfig_log_path else "Couldn't find the log file :( Check your terminal output"
+
+    commands_to_run = [
+        "which pip",
+        "which pip3",
+        "which python",
+        "which python3",
+        "python --version",
+        "python3 --version",
+    ]
+    command_inputs_outputs = dedent("\n\n".join(_fmt_command_input_output(command, _try_run_command(command)) for command in commands_to_run))
+    pip_list_output = _get_pip_list_filtered()
+    body = dedent(
+        f"""
+        This is a rage-generated bug report. Like an ordinary bug report, but with more rage.
+
+        * With markdown
+        * Cool, huh?
+
+        [TODO: detailed description]
+        [TODO: steps to reproduce (what were you doing, what did you expect to happen, what happened instead?)]
+        [TODO: server log contents ({server_log_str})]
+        [TODO: aiconfig log contents ({aiconfig_log_str})]]
+
+        =====================
+        OS: {platform.system()} {platform.release()}
+        Python: {sys.version}
+        =====================
+        {command_inputs_outputs}
+        {pip_list_output}
+
+        """
+    )
+    print("\n\n\n\nIssue draft:\n\n")
+    print(f"Title: {title}")
+    print(f"Body:\n{body}")
+    open_draft = _get_yes_or_no_input("Would you like to open a draft issue in your browser? [Y/n] ")
+    if open_draft:
+        _troll_the_user_part_2()
+        _open_github_issue_draft(
+            repo="lastmile-ai/aiconfig",
+            title=title,
+            body=body,
+            labels=["type: bug", "status: planned"],
```

Comment:
should they be 'planned' before we've looked at them? Ideally there would be a 'needs triage' bucket to put them into

Reply:
I don't think this exists, didnt want to make a new tag.